docs/extended_api/pipelines/pipeline.md

```diff
@@ -0,0 +1,155 @@
+---
+grand_parent: Extended API
+parent: Pipelines
+nav_order: 0
+---
+
+# `cuda::pipeline`
+
+```c++
+// Header `<cuda/pipeline>`.
```

Comment:
Same comment about "declared in" style as earlier.

This file needs a comment that this functionality is not a part of the github-released open-source library.